.dependency-cruiser.cjs

```diff
@@ -1,17 +1,17 @@
 /** @type {import('dependency-cruiser').IConfiguration} */
 module.exports = {
   forbidden: [
-    {
-      name: 'no-deps-for-types',
-      severity: 'error',
-      comment: `@leather-wallet/types should not have any dependencies. It should be a standalone package.`,
-      from: {
-        path: '^packages/types',
-      },
-      to: {
-        path: '^(?!packages/types)',
-      },
-    },
+    // {
+    //   name: 'no-deps-for-models',
```

Comment:
Not sure we need this now?

Reply:
I think models should be our core most package that has no deps

Reply:
This will be tough only bc we use types from Stacks in our models, guess the best thing is to then just copy them directly into our code?

Reply:
The other issue here is our use of `bignumber.js` in this package, I'm not sure how to handle that if any ideas?

Reply:
> I think it should have external dependencies

@edgarkhanzadian Which external deps might it need?

Reply:
@kyranjamie can you review the latest commit here at some point? I did only leave `bignumber.js` as a dependency and using only as a type. Proposing asset data model changes in last commit too, so hoping to get your thoughts.

Reply:
I updated the PR description bc this PR has expanded in scope.